src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -64,53 +64,18 @@ def to_internal_value(self, data):
         return data
 
 
-class RuleSerializer(serializers.Serializer):
-    name = serializers.CharField(max_length=64)
-    environment = serializers.CharField(max_length=64, required=False, allow_null=True)
+class RulePreviewSerializer(serializers.Serializer):
```

Comment:
We might call this something like `RuleConditionSerializer` (probably not that, but something like it). There's nothing that's specific to previews here, so something a bit more generic probably makes sense.

Reply:
not very good at naming, let me know if this still doesn't make sense